db/db_impl/db_impl.cc

```diff
@@ -5154,6 +5155,42 @@ Status DBImpl::TraceIteratorSeekForPrev(const uint32_t& cf_id, const Slice& key,
   return s;
 }
 
+Status DBImpl::DumpCache(const CacheDumpOptions& dump_options,
+                         const std::shared_ptr<Cache> cache) {
+  if (cache == nullptr) {
+    return Status::InvalidArgument("Cache pointer is null!");
+  }
+  CacheDumper dumper(env_, dump_options, cache, db_id_,
+                     immutable_db_options_.clock);
+  std::set<std::string> prefix_set;
+  if (db_session_id_.size() > 0) {
+    prefix_set.insert(db_session_id_);
```

Comment:
I assume this is a temporary placeholder for walking the SST files to find out the full set of prefixes they include (due to DB Open+Close, copy, etc). :)

Reply:
Yes, actually, user should directly use cache dump and load instance without DB. So, user will set the prefix filter before calls run.